packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -15,9 +14,10 @@ import {
   programmaticFocusTriggersFocusVisible,
 } from 'test/utils';
 import * as PropTypes from 'prop-types';
-import TouchRipple from './TouchRipple';
 import ButtonBase from './ButtonBase';
 
+const { expect } = require('chai').use(require('chai-style'));
```

Comment:
We are using `getComputedStyle` in these cases. Could we remove https://github.com/darlanmendonca/chai-style?